src/main/java/com/catcher/infrastructure/oauth/user/KakaoOAuthUserInfo.java

```diff
@@ -0,0 +1,26 @@
+package com.catcher.infrastructure.oauth.user;
+
+import com.catcher.core.domain.entity.enums.UserProvider;
+
+import java.util.Map;
+
+public class KakaoOAuthUserInfo extends OAuthUserInfo{
+    protected KakaoOAuthUserInfo(Map<String, Object> attributes) {
+        super(attributes);
+    }
+
+    @Override
+    public String getId() {
+        return this.getProvider().name() + "_" + attributes.get("id");
+    }
+
+    @Override
+    public String getEmail() {
+        return String.valueOf(((Map<String, Object>) attributes.get("kakao_account")).get("email"));
```

Comment:
```suggestion
        return attributes.get("kakao_account")).get("email").toString();
```
위 처럼 표현이 안되나요?
생성자에서 Map<String,Object>라서 안해도 될것 같은데 추측입니다ㅎㅎ 안될수도 있고요

Reply:
반영했습니다